source/Octopus.Manager.Tentacle.Tests/ApplicationStartupSanityCheckFixture.cs

```diff
@@ -47,13 +48,15 @@ public void ApplicationCanStartWithoutCrashing()
                 }
             });
 
+#pragma warning disable CA1416
```

Comment:
If I understand [CA1416](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1416) correctly, I'm not sure why we are ignoring this warning. What platform combination are we safeguarding against and if it does build, what do we expect the code to do when the underlying API is unavailable?

(Also applies to the other inclusions below)